core/common/src/ElementProps.ts

```diff
@@ -47,8 +47,6 @@ export interface ElementProps extends EntityProps {
   federationGuid?: GuidString;
   /** A [user-assigned label]($docs/bis/guide/fundamentals/element-fundamentals.md#userlabel) for this element. */
   userLabel?: string;
-  /** Optional [json properties]($docs/bis/guide/fundamentals/element-fundamentals.md#jsonproperties) of this element. */
```

Comment:
EntityProps define it as `{ [key: string]: any }`, which is more type-safe, so there's no need to override it here with any.